packages/bot-skeleton/src/services/api/ticks_service.js

```diff
@@ -184,30 +195,32 @@ export default class TicksService {
     }
 
     observe() {
-        const subscription = api_base.api.onMessage().subscribe(({ data }) => {
-            if (data.msg_type === 'tick') {
-                const { tick } = data;
-                const { symbol, id } = tick;
-                if (this.ticks.has(symbol)) {
-                    this.subscriptions = this.subscriptions.setIn(['tick', symbol], id);
-                    this.updateTicksAndCallListeners(symbol, updateTicks(this.ticks.get(symbol), parseTick(tick)));
+        if (api_base.api) {
+            const subscription = api_base.api.onMessage().subscribe(({ data }) => {
+                if (data.msg_type === 'tick') {
+                    const { tick } = data;
+                    const { symbol, id } = tick;
+                    if (this.ticks.has(symbol)) {
+                        this.subscriptions = this.subscriptions.setIn(['tick', symbol], id);
+                        this.updateTicksAndCallListeners(symbol, updateTicks(this.ticks.get(symbol), parseTick(tick)));
+                    }
                 }
-            }
```

Comment:
maybe it would be better to add block .catch() after the API call ?

Reply:
which place exactly?

Reply:
@vinu-deriv I've checked one more time, we have all necessary errors handlers under the hood, so everything is fine👍